hack/update-deps.sh

```diff
@@ -13,8 +13,8 @@ set -o pipefail
 cd "${ROOT_DIR}"
 
 # This controls the knative release version we track.
-KN_VERSION="release-1.7"
-EVENTING_VERSION="release-v1.7"
+KN_VERSION="release-1.8"
```

Comment:
Do we really want to biump `kn` version too, at this point? /cc @dsimansk

Reply:
KN here, amusingly, does not mean the client, it is used in various places below like 

```
  "knative.dev/pkg=knative.dev/pkg@${KN_VERSION}"
  "knative.dev/hack=knative.dev/hack@${KN_VERSION}"
  "knative.dev/networking=knative.dev/networking@${KN_VERSION}"
```

Reply:
Ah, great. LOL